pkg/snapstore/s3_snapstore.go

```diff
@@ -71,6 +75,22 @@ func newS3FromSessionOpt(bucket, prefix, tempDir string, maxParallelChunkUploads
 	return NewS3FromClient(bucket, prefix, tempDir, maxParallelChunkUploads, cli), nil
 }
 
+func getSessionOptions(prefixString string) session.Options {
+	if _, isSet := os.LookupEnv(prefixString + awsAcessKeyID); isSet {
+		return session.Options{
+			Config: aws.Config{
+				Credentials: credentials.NewStaticCredentials(os.Getenv(prefixString+awsAcessKeyID), os.Getenv(prefixString+awsSecretAccessKey), ""),
+				Region:      pointer.StringPtr(os.Getenv("AWS_REGION")),
```

Comment:
I find it strange that other `AWS_` env variable names are now extracted as constants, but `AWS_REGION` is not, so let's please extract it as well.